barretenberg/cpp/src/barretenberg/plonk_honk_shared/composer/composer_lib.test.cpp

```diff
@@ -55,7 +55,7 @@ TEST_F(ComposerLibTests, LookupReadCounts)
 
     // The table polys are constructed at the bottom of the trace, thus so to are the counts/tags
     // TODO(https://github.com/AztecProtocol/barretenberg/issues/1033): construct tables and counts at top of trace
-    size_t offset = circuit_size - builder.get_tables_size();
+    size_t offset = builder.blocks.lookup.trace_offset;
```

Comment:
I wonder if this offset should be codified into the flavor or something..

Reply:
it's a bit risky it will cause someone some annoying bugs :/ the composer lib stuff can be integrated in the decider proving key once we get rid of plonk so it would not look as odd to access trace_offsets there